marketpulse/main/urls.py

```diff
@@ -3,5 +3,6 @@
 
 
 urlpatterns = patterns(
-    url(r'^$', '{0}.views.main'.format(settings.PROJECT_NAME), name='main'),
+    '{0}.main.views'.format(settings.PROJECT_MODULE),
```

Comment:
I do this in order to avoid repeating the same url prefix, but we change that anytime in the future if it causes more trouble.